x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_indices/delete_index_modal.tsx

```diff
@@ -35,12 +36,21 @@ export const DeleteIndexModal: React.FC = () => {
       onConfirm={() => {
         deleteIndex({ indexName });
       }}
-      cancelButtonText={i18n.translate(
-        'xpack.enterpriseSearch.content.searchIndices.deleteModal.cancelButton.title',
-        {
-          defaultMessage: 'Cancel',
-        }
-      )}
+      cancelButtonText={
+        isDeleteLoading
+          ? i18n.translate(
+              'xpack.enterpriseSearch.content.searchIndices.deleteModal.closeButton.title',
+              {
+                defaultMessage: 'Close',
+              }
+            )
```

Comment:
Is this enough for translations?

Reply:
`xpack.*` seems to be untracked so is it possible we don't translate this?

Reply:
This is good enough for translations, there's no xpack-specific translation files, there's just giant translation files for everything in Kibana. English is the default message so that doesn't get a translation file.